### ISSUE TITLE
Title bar is being painted strangely

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#1023]
This is kind of weird (and might be related to the blurry issue #1017, who knows?) but I'm seeing the Title Bar and top border on PTXprint splitting apart!

![Image](https://github.com/user-attachments/assets/beac9b29-99ca-492d-859c-cea294700cdb)

Yeah, weird. It also happens on the Preview Pane: 

![Image](https://github.com/user-attachments/assets/3102ffb3-83c7-4f00-85d9-bec1d40bcfc1)

So, any time the PTXprint window gains focus, it repaints it like that. 

But if it is not in focus, it looks normal: 

![Image](https://github.com/user-attachments/assets/e67ed718-f2eb-45d3-b7c9-f9cc26cf0f28)

Just an FYI. It is not a show-stopper.

NOTE: This only happens on my external 1080p monitor set at 100% scale. It doesn't happen on my laptop screen (3840 x 2160 at 200% scale). Weirdly, I can drag the PTXpint window back and forth across the two displays, and the title bar splits and rejoins each time I do it. So, highly repeatable. It happens as soon as the window rescales as it moves across to the external monitor. (Maybe I just need to go get me a high-end 3840 x 2160 external monitor, eh? Then no jumping the scaling back and forth.)

Here it is on my laptop screen: 

![Image](https://github.com/user-attachments/assets/0956c76d-6136-4232-858c-2d0eeacc8f63)

**Comment by davidc86** [comment]
This is still an issue in 2.7.13

**Comment by markpenny** [comment]
It looks like a Gtk bug or feature. I'll keep my eyes open, but honestly, I don't think there is anything we can do about this, apart from stumble across a workaround/solution.

**Comment by davidc86** [comment]
That is odd. It didn't used to do this. Did you all change the screen painting engine for PTXprint?

**Comment by markpenny** [comment]
No, nothing has been changed that we are aware of. If you're ever testing old versions, see if you can replicated the issue there as well, or whether it is something recent (i.e. post 2.7.1) which introduced it.
